kernel/data-pipeline/cdc/client/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/client/example/Bootstrap.java

```diff
@@ -36,7 +44,18 @@ public static void main(final String[] args) {
         // Pay attention to the time zone, to avoid the problem of incorrect time zone, it is best to ensure that the time zone of the program is consistent with the time zone of the database server
         // and mysql-connector-java 5.x version will ignore serverTimezone jdbc parameter and use the default time zone in the program
         // TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
-        StartCDCClientParameter parameter = new StartCDCClientParameter(records -> log.info("records: {}", records));
+        TypeRegistry registry = TypeRegistry.newBuilder().add(EmptyProto.getDescriptor().getMessageTypes()).add(TimestampProto.getDescriptor().getMessageTypes())
+                .add(WrappersProto.getDescriptor().getMessageTypes()).build();
+        Printer printer = JsonFormat.printer().usingTypeRegistry(registry);
+        StartCDCClientParameter parameter = new StartCDCClientParameter(records -> {
+            for (Record each : records) {
+                try {
+                    log.info("record: {}", printer.print(each));
```

Comment:
Could we remove `@type`? (e.g. `"@type": "type.googleapis.com/google.protobuf.Empty"`)

Reply:
<img width="793" alt="image" src="https://user-images.githubusercontent.com/101622833/229268923-ed646f28-2ea8-4e5c-ae2c-03a71a685794.png">
seem it's necessary for JSON representation of an Any value

https://protobuf.dev/reference/protobuf/google.protobuf/